install-dependencies-dev.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Would a Windows .bat file help those users, too? We don't have one for the regular install-depenencies.sh, though. 

Similarly, should we be sure they work on macOS systems?